bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
@jsimsa , yes you are right! Would be better to we execute a *init_env $0* before *$LAUNCHER ..*